cmd/cloud/installation.go

```diff
@@ -615,3 +620,92 @@ var installationRecoveryCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var installationDeploymentReportCmd = &cobra.Command{
+	Use:   "report",
+	Short: "Get a report of deployment details for a given installation",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		installationID, _ := command.Flags().GetString("installation")
+
+		installation, err := client.GetInstallation(installationID, &model.GetInstallationRequest{
+			IncludeGroupConfig:          true,
+			IncludeGroupConfigOverrides: true,
+		})
+		if err != nil {
+			return errors.Wrap(err, "failed to query installation")
+		}
+		if installation == nil {
+			return nil
+		}
+
+		output := fmt.Sprintf("Installation: %s\n", installation.ID)
```

Comment:
I feel pretty confident that it's not worth spending the time to refactor this, but in the future for building big output strings like this one you might want to try out the [template](https://pkg.go.dev/text/template) package if you haven't before. It's super handy for building stuff like this without a million calls to `fmt.Sprintf`